quinn/src/endpoint.rs

```diff
@@ -304,6 +321,16 @@ impl Endpoint {
     }
 }
 
+// Struct for holding the stats returned from Endpoint::stats
+//
+#[derive(Debug, Default, Copy, Clone)]
+pub struct EndpointStats {
+    pub open_connections: u64,
+    pub total_incoming_handshakes: u64,
+    pub total_outgoing_handshakes: u64,
+    pub incoming_buffers_total_bytes: u64,
```

Comment:
nit: So far 3/4 of these fields have "total" in the name. Is that adding any clarity? Maybe something like "handshakes_{received, sent}" and "incoming_buffer_bytes"?